plugins/rikki/heroeslounge/classes/discord/Attendance.php

```diff
@@ -52,9 +52,49 @@ public static function FetchUsers()
       return $presentUsers;
     }
 
+    public static function fetchUserBySearch($discordTag)
+    {
+      $matchingUsers = [];
```

Comment:
nit: Try to initiate variables scoped inside a function as close to their first use as possible